src/gpu/intel/jit/v2/conv/kernel_desc.cpp

```diff
@@ -327,6 +331,23 @@ void kernel_desc_t::set_defaults() {
     }
 }
 
+void kernel_desc_t::set_stride_reqs(tensor_kind_t kind,
+        const layout_tag_t &desc_tag, prb_reqs_t &reqs) const {
+    if (desc_tag.is_strided()) {
```

Comment:
Nit: to reduce indentation:
```suggestion
    if (!desc_tag.is_strided()) return;
```